conda_smithy/configure_feedstock.py

```diff
@@ -1528,6 +1528,7 @@ def _load_forge_config(forge_dir, exclusive_config_file):
         "conda_forge_output_validation": False,
         "private_upload": False,
         "secrets": [],
+        "build_with_mambabuild": False,
```

Comment:
Can this be enabled for macOS? (and possibly Windows if boa supports it?)

Reply:
yes, it can be :) boa supports all platforms.

Reply:
I just added support for this on OS X and Win in conda-smithy.